jwst/ramp_fitting/ramp_fit_step.py

```diff
@@ -182,6 +182,7 @@ class RampFitStep(Step):
         int_name = string(default='')
         save_opt = boolean(default=False) # Save optional output
         opt_name = string(default='')
+        suppress_one_group = boolean(default=False) # Suppress ramps with only one good group
```

Comment:
The requested behavior from the Cal WG was that the slope calculation from only 1 group should NOT be used by default, so if this switch turns off the 1-group calculation, then the default value of it should be `True`, at least for now. Given the fact that `suppress_one_group=False` is kind of a case of double negatives, I wonder if it would be better to change the name and sense of the switch so that it's in the positive sense, e.g. something like `use_one_group` or `one_group_slope`. Setting that to `True` would enable the current calculation of slopes from only 1 group, while setting it to `False` would disable it. In this case the default would be `False`.

Reply:
Changed the default to be True, instead of False.